QUARKUS-728.md

```diff
@@ -7,8 +7,11 @@ The goal is to verify OpenJDK 17 integration
 ## Scope of the testing
```

Comment:
https://issues.redhat.com/browse/QUARKUS-1397 Dev preview for Java 17 in Native mode is not mentioned in JIRA links above